react-native/react/actions/tracker.js

```diff
@@ -110,11 +110,27 @@ export function onRefollow (username: string): TrackerActionCreator {
   return (dispatch, getState) => {
     console.log('onRefollow')
     const {trackToken} = (getState().tracker.trackers[username] || {})
+    const dispatchAction = () => {
+      dispatch({
+        type: Constants.onRefollow,
+        payload: {username}
+      })
+    }
+
+    dispatch(onUserTrackingLoading(username))
     trackUser(trackToken)
-    dispatch({
-      type: Constants.onRefollow,
-      payload: {username}
-    })
+      .then(dispatchAction)
+      .catch(err => {
+        console.error('Couldn\'t track user:', err)
```

Comment:
fwiw standardjs allows you to use double-quotes when the string contains single quotes, and I think that's better than escaping, just my opinion though :)